docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,480 @@
+---
+approvers:
+- mikedanese
+- luxas
+- errordeveloper
+- jbeda
+title: Creating HA clusters with kubeadm
+---
+
+{% capture overview %}
+
+This guide will instruct you on how to install and set up a highly available Kubernetes cluster using kubeadm.
+
+* The goal of this document is to cover the areas that kubeadm was not designed to cover: the provisioning of hardware, configuration of multiple systems, and load balancing. The user is expected to handle this themselves with the aid of our documentation. 
+* Please note this guide is one potential solution - there are many ways to configure HA! If a better solution works for you, please use it. If you feel it can be adopted by the community, feel free to contribute it back.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+1. Three machines running Ubuntu 16.04+, CentOS 7 or HypriotOS v1.0.1+ for the masters. 
+1. Three machines running Ubuntu 16.04+, CentOS 7 or HypriotOS v1.0.1+ for the workers. 
+1. (optional) At least three machines if you intend to host etcd on dedicated nodes (see information below).
+1. 1GB or more of RAM per machine (any less will leave little room for your apps).
+1. Full network connectivity between all machines in the cluster (public or
+   private network is fine).
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Set up masters
```

Comment:
What about moving this just after "Set up master Load Balancer" paragraph, so we have everything related to the etcd cluster first, the load balancer, and then everything related to master nodes?

Reply:
If users want to host etcd on the masters using static pods, they need the kubelet installed. So that's why I put this section here. I agree having it near the LB section would make the most sense otherwise